src/etc/maketest.py

```diff
@@ -53,6 +53,8 @@ def convert_path_spec(name, value):
 putenv('TARGET_RPATH_DIR', os.path.abspath(sys.argv[11]))
 putenv('RUST_BUILD_STAGE', sys.argv[12])
 putenv('S', os.path.abspath(sys.argv[13]))
+putenv('RUSTFLAGS', sys.argv[15])
```

Comment:
Was this necessary to pass along? It looks like this isn't used elsewhere from this patch? Also this looks like it skipped a number?